source/reference/mongosync.txt

```diff
@@ -1,9 +1,95 @@
 .. _c2c-mongosync:
 
+.. default-domain:: mongodb
+
+.. program:: mongosync
+
 =============
 ``mongosync``
 =============
 
-.. default-domain:: mongodb
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Synopsis
```

Comment:
The other docs typically use "Description" instead of "Synopsis"

Reply:
Fixed